### ISSUE TITLE
Admin VPN DNS does not resolve hosts outside of the HiSPARC VPN

### ISSUE THREAD

**Issue by tomkooij** [HiSPARC/publicdb#251]
When connected to the HiSPARC VPN we use `tietar` as DNS (to resolve `*.his`). This DNS should also resolve hosts outside of the VPN. 

This broke when we changed to the new tietar.

**Comment by tomkooij** [comment]
I have added `-q` to dnsmasq (to show logs in `/var/log/messages` (only temporarily, because logs are HUGE)

When I query `xs4all.nl` when connected to admin VPN (I am `172.16.66.2`): 
```
Jan 20 19:20:22 tietar dnsmasq[21011]: query[A] xs4all.nl from 172.16.66.2
Jan 20 19:20:22 tietar dnsmasq[21011]: forwarded xs4all.nl to 192.16.199.17
Jan 20 19:20:22 tietar dnsmasq[21011]: forwarded xs4all.nl to 192.16.185.12
```
So the request is forwarded to to `madon` and `cesano`. Both are DNS servers at Nikhef.

**Comment by tomkooij** [comment]
The Nikhef DNS servers are indeed blocked from within the DMZ 👎 

I changed `/etc/resolv.conf-nikhef` to:
```
nameserver 8.8.8.8
```

and now:
```

Jan 20 19:27:19 tietar dnsmasq[22278]: query[A] xs4all.nl from 172.16.66.2
Jan 20 19:27:19 tietar dnsmasq[22278]: forwarded xs4all.nl to 8.8.8.8
Jan 20 19:27:19 tietar dnsmasq[22278]: reply xs4all.nl is 194.109.6.93
Jan 20 19:27:20 tietar dnsmasq[22278]: query[AAAA] xs4all.nl from 172.16.66.2
Jan 20 19:27:20 tietar dnsmasq[22278]: forwarded xs4all.nl to 8.8.8.8
Jan 20 19:27:20 tietar dnsmasq[22278]: reply xs4all.nl is 2001:888:0:18::93
```

It works!

I'll push a fix.

**Comment by tomkooij** [comment]
Fixed in c2a7b8df0698622c1e456b8cd4cfc66492762707

**Comment by 153957** [comment]
Unfortunate that 1.1.1.1 also does not work, but nice troubleshooting and fix!
